oraclepaas/resource_mysql_service_instance_test.go

```diff
@@ -0,0 +1,235 @@
+	backup_destination                   = "BOTH"
+	
+	cloud_storage_configuration {
+		cloud_storage_container = "%s"
+		create_if_missing = true				
+	}
+			
+	mysql_configuration = {
+		db_name                              = "demo_db"
+		db_storage                           = 25
+		mysql_port                           = 3306
+		mysql_username                       = "root"
+		mysql_password                       = "MySqlPassword_1"
+		shape                                = "oc3"
+		mysql_charset                        = "utf8"
+		mysql_collation                      = "utf8_general_ci"
+	    enterprise_monitor                   = false		
+	}
+}`, rInt, container)
+}
+
+func testAccMySQLServiceInstanceOCI(rInt int) string {
+
+	return fmt.Sprintf(`
+resource "oraclepaas_mysql_service_instance" "test" {
+		
+	service_description                  = "test-service-instance"
+	service_name                         = "TestInst%d"
+	ssh_public_key                       = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC0Pspsfu8lUTxILGf+dJnTTbIeFZrL/NKaQNNEvH9jF9aXcr347C5dKlu45LE2jTB8OfjtaExOznn7kKiOErwWPJUzDncDDsmUacDzs5KGbDBGQb6zxEMyYgYCKDiru5V24CrZqam+3QP5AurLopD3JaYmZSikKgP+syu16jBs3WzRLvGzDknIkrUk6t7XjzJ5X/wgMTqepjDDyn9NJ3nG5l4iQe7ULgAbfnRjTM3pRQZ5EM67iN3jc+cIFeNsEwqnxb9ZCJ7avb+Yqdcm/7A5tlX+rMwnTYYCPF/j8bgFdHuO9VHEiQHkM7FuRvZGWkXCryyg9iLM+myG5XdVa3Z2IsfBx3qIfxKMcWsHIk5mmDvWIDbgvBne6JSPKhkB7qM6F10pJSVvt08tGwmlTxZZJPKCkpd0nrfrVChMdMr9yRoYH46bqwMbPFCffNeVkJfj4IMlSSU+A9RGLLEnkdv+Xk3yCS+8RcNA6Zilv9VnJm4hBEJ2LsDVZfwqTvUAeB4evpOCMS+v4YKn/w+R4cB/+SdYDtifBwKW8TYk4ZK3J4wHa6XAI4u3b9C0bIfUmXZs36Gyy4MArtg6QGqrmTzYMa5eI2uB7BnO0JM/Moref8vvQYvGjbnkC5G/yCoLswbt477Gn+Ih96PyZ81qMmTv8qE9S3F3qCqkR3sDJA3oDw=="
+	backup_destination                   = "NONE"
+	region                               = "%s"
+	availability_domain                  = "%s"
+
+	mysql_configuration = {
+		db_name                              = "demo_db"
+		db_storage                           = 25
+		mysql_port                           = 3306
+		mysql_username                       = "root"
+		mysql_password                       = "MySqlPassword_1"
+		shape                                = "oc3"
+		mysql_charset                        = "utf8"
+		mysql_collation                      = "utf8_general_ci"
+	    enterprise_monitor                   = false		
+		subnet                               = "%s"
+	}
+}`, rInt, oci_region, oci_availability_domain, oci_subnet)
```

Comment:
As noted above, let's read these from the environment

Reply:
Done